Cargo.toml

```diff
@@ -7,6 +7,7 @@ readme = "README.md"
 repository = "https://github.com/sonatype-nexus-community/cargo-pants"
 description = "cargo-pants is a cargo subcommand application that provides a bill of materials and a list of which dependencies have a vulnerability, powered by Sonatype OSSIndex"
 license = "Apache-2.0"
+rust = "1.53"
```

Comment:
`cargo check` gives

> warning: unused manifest key: package.rust

Are we concerned about using a cargo feature that is still in the RFC stage, given that the key is an open question? [RFC context here](https://github.com/rust-lang/rust/issues/65262)

Reply:
Interesting to know. I'm not concerned. "What, me worry?"